package.json

```diff
@@ -22,6 +22,7 @@
     "cypress:run": "./node_modules/.bin/cypress run",
     "cypress": "./node_modules/.bin/cypress open",
     "dead-code": "./scripts/check-dead-code.sh",
+    "dead-code:production": "./scripts/check-dead-code.sh production",
```

Comment:
Any chance we can make this more self-describing? Having a hard time coming up with something adequate tho...

Reply:
Yeah I struggled with that, couldn't think of a better name, and so kept it 1:1 with the Knip concept

Reply:
I almost like this as the default, if something is only used in tests that's a ✂️ as far as I'm concerned!